fact_extractor/plugins/unpacking/dlink_shrs/internal/DecryptorLink.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import hashlib
+import sys
+import binascii
+import pathlib
+
+from Crypto.Cipher import AES
+
+
+class DcryptLink:
+
+    def __init__(self):
+        self.enc_fw = sys.argv[1]
```

Comment:
this could be accomplished a bit less ugly with `argparse` but since this script is only called by the plugin this could be overkill